packages/components/src/components/Box/Box.tsx

```diff
@@ -10,6 +10,7 @@ import { TransientProps } from '../../utils/transientProps';
 
 export const allowedBoxFrameProps = [
     'margin',
+    'padding',
```

Comment:
Is it necesary to have this in `Box` since we have `margin`?

Reply:
Yep, there are cases when padding behaves differently to margin (for example, margin collapse etc)